src/components/Schedule/ProgressBar.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from "react";
+
+interface IProps {
+  currentReservation: number;
+  maxReservation: number;
+}
+
+export const ProgressBar: FC<IProps> = ({
+  currentReservation,
+  maxReservation
+}) => (
+  <div>
```

Comment:
we don't need this div. wrap in `<>` . if not sure about the syntax, read https://reactjs.org/docs/fragments.html#short-syntax